api_tabular/metrics.py

```diff
@@ -52,6 +52,7 @@ async def get_object_data_streamed(
                 handle_exception(res.status, "Database error", await res.json(), None)
             async for chunk in res.content.iter_chunked(1024):
                 yield chunk
+                yield b'\n'
```

Comment:
Shouldn't it we yield \n at the end of `iter_chunked` only?
```suggestion
            yield b'\n'
```